src/features/workspace/components/workspace-custom-instructions.tsx

```diff
@@ -33,6 +46,9 @@ import {
 import { v1GetWorkspaceCustomInstructionsQueryKey } from "@/api/generated/@tanstack/react-query.gen";
 import { useQueryGetWorkspaceCustomInstructions } from "../hooks/use-query-get-workspace-custom-instructions";
 import { useMutationSetWorkspaceCustomInstructions } from "../hooks/use-mutation-set-workspace-custom-instructions";
+import { Bot, Download, Search } from "lucide-react";
```

Comment:
I guess we can use untitledui/icons now that we have ui-kit, or I mean let's do a clean about icons in a separate PR too, it seems we are using both lucide and untitledui

Reply:
yeah, separate PR to replace all icons would make sense